Sources/ComposableCoreLocation/Models/Location.swift

```diff
@@ -36,20 +35,16 @@ public struct Location {
   }
 }
 
-extension Location: Equatable {
+extension Location: Hashable {
   public static func == (lhs: Self, rhs: Self) -> Bool {
-    let speedAccuracyIsEqual: Bool
     let courseAccuracyIsEqual: Bool
-
     #if compiler(>=5.2)
       if #available(iOS 13.4, macCatalyst 13.4, macOS 10.15.4, tvOS 13.4, watchOS 6.2, *) {
         courseAccuracyIsEqual = lhs.courseAccuracy == rhs.courseAccuracy
       } else {
         courseAccuracyIsEqual = true
       }
-      speedAccuracyIsEqual = lhs.speedAccuracy == rhs.speedAccuracy
```

Comment:
I think this didn't need to be guarded in the `#if`, seems perfectly available on all the platforms/OS's we support

Reply:
Hm tests are failing with

> Value of type 'CLLocation' has no member 'speedAccuracy'

Reply:
huh strange, docs say `speedAccuracy` was introduced in iOS 10.0, but maybe that is wrong. will change this bit.